gui/src/2D/controls/displayGrid.ts

```diff
@@ -183,7 +183,7 @@ export class DisplayGrid extends Control {
                 context.strokeStyle = this._minorLineColor;
                 context.lineWidth = this._minorLineTickness;
 
-                for (var x = -cellCountX / 2; x < cellCountX / 2; x++) {
+                for (var x = -cellCountX / 2 + 1; x < cellCountX / 2; x++) {
                     const cellX = left + x * this.cellWidth;

```

Comment:
doesn't that depend on the number of cells (i.e. on cellCountX and cellCountY)? or better - whether or not these numbers are odd/even?

Reply:
I can test with even different numbers too. I thought I was just increasing starting the iterations 1 forward.

Reply:
![image](https://user-images.githubusercontent.com/67929198/131540063-1f4e585d-c473-4241-a73c-bfdf0f20a0b0.png)
 
Tested out some wacky cell w/h and grid w/h and looks good to me.